cli/util/draw_thread.rs

```diff
@@ -30,7 +30,6 @@ impl Drop for DrawThreadGuard {
 
 #[derive(Debug, Clone)]
 struct InternalEntry {
-  priority: u8,
```

Comment:
Removed because we're not using this functionality. I investigated a bit at making progress bars show while displaying the permission prompt and it would be too complicated.